src/main/java/org/cactoos/list/Immutable.java

```diff
@@ -198,21 +248,52 @@ public int lastIndexOf(final Object item) {
 
     @Override
     public List<T> subList(final int start, final int end) {
-        return this.list.subList(start, end);
+        return new Immutable<>(
+            this.list.subList(start, end)
+        );
     }
 
     @Override
-    public String toString() {
-        return this.list.toString();
+    @edu.umd.cs.findbugs.annotations.SuppressFBWarnings("EQ_UNUSUAL")
```

Comment:
@iakunin is possible change `@edu.umd.cs.findbugs.annotations.SuppressFBWarnings("EQ_UNUSUAL")` for `@SuppressFBWarnings("EQ_UNUSUAL")`?

Reply:
@fabriciofx fixed